tests/countries/test_cambodia.py

```diff
@@ -19,7 +20,7 @@
 class TestCambodia(CommonCountryTests, TestCase):
     @classmethod
     def setUpClass(cls):
-        super().setUpClass(Cambodia, years=range(1993, 2059))
+        super().setUpClass(Cambodia, years=chain(range(1993, 2059), [2158]))
```

Comment:
Would list + list work here and below w/o the `chain` import?
```suggestion
        super().setUpClass(Cambodia, years=list(range(1993, 2059)) + [2158]
```

Reply:
Fixed.